nupic/data/utils.py

```diff
@@ -147,3 +147,16 @@ def serializeSdr(sdr):
   """
 
   return "".join(str(bit) for bit in sdr)
+
+
+
+def parseStringList(s):
+  """Parse a string of space-separated numbers, returning a Python list."""
+  assert isinstance(s, basestring)
+  return [int(i) for i in s.split()]
+
+
+
+def stripList(listObj):
+  """Convert a list of numbers to a string of space-separated numbers."""
+  return string.replace(str(listObj)[1:-1], ",", "")
```

Comment:
First, you can use `replace` as a method of a string rather than the function in the `string` module.

But even better would be:

```
return " ".join(str(i) for i in listObj)
```